src/components/FAQ/FaqContent.tsx

```diff
@@ -3,19 +3,30 @@ import ReactMarkdown from "react-markdown";
 import { AccordionItem } from "../../components/UI/Accordion/AccordionItem";
 import { importAll } from "../../common/utils/importAll";
 import { getSortedByDateAsc } from "../../utils";
+import { FaqType } from "../../constants";
 
 export type FAQS = {
   body: string;
   attributes: {
     title: string;
+    type: FaqType;
   };
 };
 
-let faqs = importAll(require.context("../../../cms/faq/", false, /\.md$/)) as FAQS[];
-faqs = getSortedByDateAsc(faqs);
+interface FAQ {
+  faqType: FaqType;
+}
 
-export const FaqContent: FunctionComponent = () => {
+export const FaqContent: FunctionComponent<FAQ> = ({ faqType }) => {
   const [openIndex, setOpenIndex] = useState(-1);
+
+  const allFaqs = importAll(require.context("../../../cms/faq/", false, /\.md$/)) as FAQS[];
+
+  let faqs = allFaqs.filter((faq) => {
+    return faq.attributes.type === faqType;
+  });
+  faqs = getSortedByDateAsc(faqs);
```

Comment:
do these in a hook?
o/w they will keep on executing these steps everytime the render() funciton is called.

Reply:
ok